var/spack/repos/builtin/packages/py-fenics-dolfinx/package.py

```diff
@@ -30,34 +33,42 @@ class PyFenicsDolfinx(PythonPackage):
     depends_on("python@3.7:3.10", when="@0.4", type=("build", "run"))
 
     depends_on("fenics-dolfinx@main", when="@main")
+    depends_on("fenics-dolfinx@0.7.2", when="@0.7.2")
+    depends_on("fenics-dolfinx@0.7.1", when="@0.7.1")
+    depends_on("fenics-dolfinx@0.7.0", when="@0.7.0")
     depends_on("fenics-dolfinx@0.6.0", when="@0.6.0")
     depends_on("fenics-dolfinx@0.5.1", when="@0.5.1")
     depends_on("fenics-dolfinx@0.5.0", when="@0.5.0")
     depends_on("fenics-dolfinx@0.4.1", when="@0.4.1")
 
     depends_on("fenics-basix@main", type=("build", "link"), when="@main")
+    depends_on("fenics-basix@0.7.0:0.7", type=("build", "link"), when="@0.7.0:0.7")
```

Comment:
I don't see this in `python/setup.py`, is that a bug in the Spack recipe (package.py) or a bug in the pip recipe (setup.py)?